File: sample/src/main/java/contacts/sample/cheatsheet/accounts/java/QueryAccountsActivity.java
package contacts.sample.cheatsheet.accounts.java;

import android.accounts.Account;
import android.app.Activity;

import java.util.List;

import contacts.core.ContactsFactory;

public class QueryAccountsActivity extends Activity {

    List<Account> getAllAccounts() {
        return ContactsFactory.create(this).accounts().query().find();
    }

    List<Account> getAllGoogleAccounts() {
        return ContactsFactory.create(this)
                .accounts()
                .query()
                .withTypes("com.google")
                .find();
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/accounts/kotlin/QueryAccountsActivity.kt
package contacts.sample.cheatsheet.accounts.kotlin

import android.accounts.Account
import android.app.Activity
import contacts.core.Contacts

class QueryAccountsActivity : Activity() {

    fun getAllAccounts(): List<Account> = Contacts(this).accounts().query().find()

    fun getAllGoogleAccounts(): List<Account> = Contacts(this)
        .accounts()
        .query()
        .withTypes("com.google")
        .find()
}